test/tunicode.c

```diff
@@ -319,12 +352,30 @@ test_utf8_conv_failure(void)
 static void
 test_code_page_override(void)
 {
-    const char *filename = "\xc3\x9f"; /* UTF-8 'ß' / Windows code page 1252 'ÃŸ' */
+    const char *filename = "\xc3\x9f"
+                           ".h5"; /* UTF-8 'ß' / Windows code page 1252 'ÃŸ' */
     hid_t       fid      = H5I_INVALID_HID;
+    hid_t       fapl     = H5I_INVALID_HID;
     char       *env      = NULL;
+    char        fixed_filename[128];
```

Comment:
Might want to move the '128' to a #define.

Reply:
Easy enough. FYI we dismiss approvals automatically now on changes to PRs.